docs/providers/google/web.md

```
@@ -0,0 +1,202 @@
+<script setup lang="ts">
+import Picture from '../../components/Picture.vue';
+</script>
+
+### Using Google login on the web
+
+Using the google login on the web is rather simple.
+In order to use it, you have to do the following:
+
+1. Create a web client in the Google Console.
+   
+  - Please follow step 2 of the [Using Google login on Android](./android.  md#_2-create-a-web-client-this-is-required-for-android) if you have not configured it already
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)